### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by yhzcake** [BakaXL-Support/BakaXL-QA-docs#129] [使用时的疑问] [answered]
![image](https://github.com/BakaXL-Support/BakaXL-QA-docs/assets/80627671/ad9fc126-9f63-4925-a5f9-4f944c24f4b8)

**Comment by XiaoMouz** [comment, marked as answer]
在hosts文件添加 
```
local.baka.zone 127.0.0.1
```
